--- dwarf-20160115/libdwarf/Makefile.in.orig	2016-04-05 17:13:20.829368602 +0200
+++ dwarf-20160115/libdwarf/Makefile.in	2016-04-05 17:18:41.089355162 +0200
@@ -129,7 +129,7 @@
 	$(AR) $(ARFLAGS) $@ $(OBJS)  dwarf_names.o > ar-output-temp
 
 libdwarf.so: dwarf_names.h dwarf_names.c $(OBJS) dwarf_names.o
-	$(CC) $(CFLAGS) -shared $(OBJS)  dwarf_names.o $(dwfzlib) -o $@
+	$(CC) $(CFLAGS) -shared -Wl,-soname,libdwarf.so.0 $(OBJS)  dwarf_names.o $(dwfzlib) -o $@ -lelf
 
 none:
 	echo "do nothing " $@
