--- dwarf-20160613/libdwarf/Makefile.in.orig	2016-06-08 17:26:52.000000000 +0200
+++ dwarf-20160613/libdwarf/Makefile.in	2016-07-31 21:21:57.395979934 +0200
@@ -132,7 +132,7 @@
 	$(AR) $(ARFLAGS) $@ $(OBJS)  dwarf_names.o > ar-output-temp
 
 libdwarf.so: dwarf_names.h dwarf_names.c $(OBJS) dwarf_names.o
-	$(CC) $(CFLAGS) $(LDFLAGS) -shared $(OBJS) -Wl,-soname=$(SONAME)  dwarf_names.o $(dwfzlib) -o $@
+	$(CC) $(CFLAGS) $(LDFLAGS) -shared $(OBJS) -Wl,-soname=$(SONAME)  dwarf_names.o $(dwfzlib) -o $@ -lelf
 	ln libdwarf.so $(SONAME)
 
 none:
