--- dwarf-20150915/dwarfdump/Makefile.in.orig	2015-09-15 18:36:08.000000000 +0200
+++ dwarf-20150915/dwarfdump/Makefile.in	2015-10-24 22:55:36.597568219 +0200
@@ -12,7 +12,7 @@
 prefix =	@prefix@
 exec_prefix =	@exec_prefix@
 bindir =	$(exec_prefix)/bin
-libdir =	$(exec_prefix)/lib
+sysconfdir =	@sysconfdir@
 mandir =        $(exec_prefix)/share/man
 man1dir =       $(mandir)/man1
 
@@ -33,8 +33,8 @@
 DIRINC =        $(srcdir)/../libdwarf
 LIBS =		@LIBS@ -L../libdwarf -ldwarf -lelf
 INCLUDES =	-I. -I$(srcdir) -I$(srcdir)/../libdwarf
-CFLAGS =	$(PREINCS) @CFLAGS@ $(INCLUDES) -DCONFPREFIX=${libdir} $(POSTINCS)
-LDFLAGS =	$(PRELIBS) @LDFLAGS@  $(LIBS) $(POSTLIBS)
+CFLAGS =	$(PREINCS) @CFLAGS@ $(INCLUDES) -DCONFPREFIX=${sysconfdir} $(POSTINCS)
+LDFLAGS =	$(PRELIBS) @LDFLAGS@ -Wl,-rpath,../libdwarf $(LIBS) $(POSTLIBS)
 #VG =            valgrind --leak-check=full
 VG =
 
@@ -191,9 +191,9 @@
 # will work and leave sensible permissions on the resulting files.
 # Some adjustment might be required, see README.
 install: all
-	$(INSTALL) dwarfdump $(bindir)/dwarfdump
-	$(INSTALL) $(srcdir)/dwarfdump.conf $(libdir)/dwarfdump.conf
-	$(INSTALL) $(srcdir)/dwarfdump.1 $(man1dir)/dwarfdump.1
+	$(INSTALL) dwarfdump $(DESTDIR)$(bindir)/dwarfdump
+	$(INSTALL) $(srcdir)/dwarfdump.conf $(DESTDIR)$(sysconfdir)/dwarfdump.conf
+	$(INSTALL) $(srcdir)/dwarfdump.1 $(DESTDIR)$(man1dir)/dwarfdump.1
 
 uninstall:
 	-rm -f $(bindir)/dwarfdump
