--- dwarf-20161124/dwarfdump/Makefile.in.orig	2016-11-24 22:14:30.000000000 +0100
+++ dwarf-20161124/dwarfdump/Makefile.in	2017-04-09 18:40:55.035853286 +0200
@@ -11,7 +11,7 @@
 prefix =	@prefix@
 exec_prefix =	@exec_prefix@
 bindir =	$(exec_prefix)/bin
-libdir =	$(exec_prefix)/lib
+sysconfdir =	@sysconfdir@
 mandir =        $(exec_prefix)/share/man
 man1dir =       $(mandir)/man1
 
@@ -36,8 +36,8 @@
 DIRINC =        $(srcdir)/../libdwarf
 LIBS =		@LIBS@ -L../libdwarf -ldwarf -lelf $(dwfzlib)
 INCLUDES =	-I. -I$(srcdir) -I../libdwarf -I$(srcdir)/../libdwarf
-CFLAGS =	$(PREINCS) @CPPFLAGS@ @CFLAGS@ $(INCLUDES) $(dwfwall) $(dwfsanitize) -DCONFPREFIX=${libdir} $(POSTINCS)
-LDFLAGS =	$(PRELIBS) @LDFLAGS@  $(LIBS) $(dwfsanitize) $(POSTLIBS)
+CFLAGS =	$(PREINCS) @CPPFLAGS@ @CFLAGS@ $(INCLUDES) $(dwfwall) $(dwfsanitize) -DCONFPREFIX=${sysconfdir} $(POSTINCS)
+LDFLAGS =	$(PRELIBS) @LDFLAGS@ -Wl,-rpath,../libdwarf $(LIBS) $(dwfsanitize) $(POSTLIBS)
 #VG =            valgrind --leak-check=full  --show-leak-kinds=all
 VG =
 
@@ -201,9 +201,9 @@
 # will work and leave sensible permissions on the resulting files.
 # Some adjustment might be required, see README.
 install: all
-	$(INSTALL) dwarfdump $(bindir)/dwarfdump
-	$(INSTALL) $(srcdir)/dwarfdump.conf $(libdir)/dwarfdump.conf
-	$(INSTALL) $(srcdir)/dwarfdump.1 $(man1dir)/dwarfdump.1
+	$(INSTALL) dwarfdump $(DESTDIR)$(bindir)/dwarfdump
+	$(INSTALL) $(srcdir)/dwarfdump.conf $(DESTDIR)$(sysconfdir)/dwarfdump.conf
+	$(INSTALL) $(srcdir)/dwarfdump.1 $(DESTDIR)$(man1dir)/dwarfdump.1
 
 uninstall:
 	-rm -f $(bindir)/dwarfdump
