--- dwarf-20160613/dwarfdump/Makefile.in.orig	2016-06-03 22:12:37.000000000 +0200
+++ dwarf-20160613/dwarfdump/Makefile.in	2016-07-31 21:01:28.765993963 +0200
@@ -11,7 +11,7 @@
 prefix =	@prefix@
 exec_prefix =	@exec_prefix@
 bindir =	$(exec_prefix)/bin
-libdir =	$(exec_prefix)/lib
+sysconfdir =	@sysconfdir@
 mandir =        $(exec_prefix)/share/man
 man1dir =       $(mandir)/man1
 
@@ -35,8 +35,8 @@
 DIRINC =        $(srcdir)/../libdwarf
 LIBS =		@LIBS@ -L../libdwarf -ldwarf -lelf $(dwfzlib)
 INCLUDES =	-I. -I$(srcdir) -I../libdwarf -I$(srcdir)/../libdwarf
-CFLAGS =	$(PREINCS) @CPPFLAGS@ @CFLAGS@ $(INCLUDES) $(dwfwall) -DCONFPREFIX=${libdir} $(POSTINCS)
-LDFLAGS =	$(PRELIBS) @LDFLAGS@  $(LIBS) $(POSTLIBS)
+CFLAGS =	$(PREINCS) @CPPFLAGS@ @CFLAGS@ $(INCLUDES) $(dwfwall) -DCONFPREFIX=${sysconfdir} $(POSTINCS)
+LDFLAGS =	$(PRELIBS) @LDFLAGS@ -Wl,-rpath,../libdwarf $(LIBS) $(POSTLIBS)
 #VG =            valgrind --leak-check=full
 VG =
 
@@ -194,9 +194,9 @@
 # will work and leave sensible permissions on the resulting files.
 # Some adjustment might be required, see README.
 install: all
-	$(INSTALL) dwarfdump $(bindir)/dwarfdump
-	$(INSTALL) $(srcdir)/dwarfdump.conf $(libdir)/dwarfdump.conf
-	$(INSTALL) $(srcdir)/dwarfdump.1 $(man1dir)/dwarfdump.1
+	$(INSTALL) dwarfdump $(DESTDIR)$(bindir)/dwarfdump
+	$(INSTALL) $(srcdir)/dwarfdump.conf $(DESTDIR)$(sysconfdir)/dwarfdump.conf
+	$(INSTALL) $(srcdir)/dwarfdump.1 $(DESTDIR)$(man1dir)/dwarfdump.1
 
 uninstall:
 	-rm -f $(bindir)/dwarfdump
